docs/ldml/tr35-numbers.md

```diff
@@ -1315,6 +1321,73 @@ Here is a set of heuristic rules that may be helpful:
+
+If the number is negative, or if the formatting options request the sign to be displayed, *prepend* the `approximatelySign` to the plus or minus sign before substituting it into the pattern. For example, "\~-5" means "approximately negative five". This procedure may change in the future.
+
+### 8.2 <a name="Collapsing_Number_Ranges" href="#Collapsing_Number_Ranges">Collapsing Number Ranges</a>
+
+*Collapsing* a number range refers to the process of removing duplicated information in the *lower* and *upper* values. For example, if the lower string is "3.2 centimeters" and the upper string is "4.5 centimeters", it is desirable to remove the extra "centimeters" token.
+
+This operation requires *semantic annotations* on the formatted value. The exact form of the semantic annotations is implementation-dependent. However, implementations may consider the following broad categories of tokens:
+
+1. Numerical value, including decimal and grouping separators
+1. Sign symbol
+1. Scientific or compact notation
+1. Unit of measurement
+
+For example, consider the string `-5.3M US dollars`. It may be annotated as follows:
+
+- `-` → sign symbol
+- `5.3` → numerical value
+- `M` → compact notation
+- `US dollars` → unit of measurement for the currency USD
+
+Two tokens are *semantically equivalent* if they have the same *semantic annotations*, even if they are not the exact same string. For example:
+
+1. "centimeter" is semantically equivalent to "centimeters".
+1. "K" (the thousands symbol in compact decimals) is NOT semantically equivalent to "K" (the measurement unit Kelvin).
+
+Two semantically equivalent tokens can be *collapsed* if they appear at the start of both values or the end of both values. However, the implementation may choose different levels of aggressiveness with regard to collapsing tokens.
+
+**Heuristic:** Only collapse semantically equivalent *unit of measurement* tokens, and only if the tokens are more than one code point in length. This is to avoid ambiguous strings such as "3–5K" (could represent 3–5000 or 3000–5000).
+
+**To collapse tokens:** Remove the token from both values, and then re-compute the token based on the number range. If the token depends on the plural form, follow [Plural Ranges](#Plural_Ranges) to calculate the correct form. If the tokens originated at the beginning of the string, prepend the new token to the beginning of the *lower* string; otherwise, append the new token to the end of the *upper* string.
+
+### 8.3 <a name="Range_Pattern_Processing" href="#Range_Pattern_Processing">Range Pattern Processing</a>
+
+To obtain a number range pattern, the following steps are taken:
+
+1. Load the range pattern found in [Miscellaneous Patterns](#Miscellaneous_Patterns).
+1. Optionally add spacing to the range pattern.
+
+**Heuristic:** If the *lower* string ends with a character other than a digit, or if the *upper* string begins with a character other than a digit, add spacing to the range pattern.
```

Comment:
Change to:

The current recommended heuristic is:

 * If the lower string ends with a character other than a digit or if the upper string begins with a character other than a digit
 * **and** if the range pattern does not contain a character having the White_Space binary Unicode property after the {0} or before the {1} placeholders
 * then insert a non-breaking space (U+00A0) at those positions.

These heuristics may be refined in the future.

[[Note change of 0020 to 00A0]]

Reply:
done